React/AccessibilityResources/en.lproj/Localizable.strings

```diff
@@ -24,3 +24,4 @@
 "expanded"="expanded";
 "collapsed"="collapsed";
 "mixed"="mixed";
+"table"="table";
```

Comment:
why do we need this loc string? we don't localize anything in this repo do we?

Reply:
This is also in core. I'm not sure what these do since afaik they're never localized. I'd prefer we keep our addition though

Reply:
^If impact is unknown, is it worth the diff..?

Reply:
i rather not have this change if there is no use of it.

Reply:
So I followed up with Saad on this offline, seems like there's impact in adding this change (see https://github.com/facebook/react-native/pull/30689). Since there's impact, I'm going to keep and add a diff tag instead of upstreaming it since this is macOS specific.

Reply:
if we keep this,  how are we going to localize this file for mac?

Reply:
Turns out this file doesn't localize strings for macOS. Also created #1359 for followup